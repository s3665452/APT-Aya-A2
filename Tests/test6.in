UURUYBLRUBBLLRRRBYUYBRRRLUBRBBUULLLYBBRULBYYRYUULURYYYLURYYUYBLLLBLRRRUBLYURYLBURYYLRYULURLBYYBUBLBB
Chris
Amy
turn 1 U 3
turn 4 R 3
turn 5 Y 2
turn 3 B 2
turn 2 L 1