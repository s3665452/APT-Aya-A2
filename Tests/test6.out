Factories:
0: R L B U U L Y B R
1: 
2: 
3:
4: 
5: 

Score for Chris: 0
Mosaic for Chris:
1:         L || . . . . .
2:       Y Y || . . . . .
3:     U U U || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken:

Score for Amy: 0
Mosaic for Amy:
1:         . || . . . . .
2:       B B || . . . . .
3:     R R R || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken: F
